### PR TITLE
Pubsub: use custom fs2-google-pubsub-grpc dependency

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -69,7 +69,7 @@ object Dependencies {
     val decline          = "1.0.0"
     val fs2              = "2.5.5"
     val catsEffect       = "2.5.0"
-    val fs2PubSub        = "0.16.1"
+    val fs2PubSub        = "0.18.0-M8"
```
Comment (benjben) [inline project/Dependencies.scala:72]: Why don't we release final version?

**Review by benjben — APPROVED** [review]
:+1:

File: project/Dependencies.scala (hunk 1 above)
Comment (oguzhanunlu) [reply project/Dependencies.scala]: just addressed

**Comment by benjben** [comment]
See https://github.com/snowplow/enrich/issues/487#issuecomment-1021012957

**Pull request closed without merging**